sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/SarifParserCallbackImpl.java

```diff
@@ -53,6 +54,7 @@
   private static final Logger LOG = Loggers.get(SarifParserCallbackImpl.class);
 
   private static final String EXTERNAL_ENGINE_ID = "roslyn";
+  private static final ImmutableList<String> OWN_REPOSITORIES = ImmutableList.of("csharpsquid", "vbnet");
```

Comment:
_minor_ : as in c#, it's better to use the top level interface for declaration instead of the implementing class
```suggestion
  private static final List<String> OWN_REPOSITORIES = ImmutableList.of("csharpsquid", "vbnet");
```

Reply:
I had to change this in the end to `List<String> OWN_REPOSITORIES =  Arrays.asList("csharpsquid", "vbnet")` because the integration tests were failing due to `java.lang.NoClassDefFoundError: org/sonar/api/internal/google/common/collect/ImmutableList`.